### PR TITLE
feat/card-posts

### PR DESCRIPTION
**Pull request by Miglemeta**
feat/cards-posts

### CODE REVIEW

File: src/App_BACKUP_1212.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  list-style: none;
+}
+
+a, h3 {
+  color: darkslateblue;
+}
+<<<<<<< Updated upstream
```
Comment (adtm) [inline src/App_BACKUP_1212.css]: merge conflict probably.

File: src/App_BACKUP_1212.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
+.App {
```
Comment (adtm) [inline src/App_BACKUP_1212.css]: I'm not sure you want to commit this, looking at the file name.

Comment (adtm) [reply src/App_BACKUP_1212.css]: I see there are many of these :)

File: src/Card.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+
+function Card (props) {
+    return (
+        <div className="card">
+            <div className="left-column">
+            <img className="thumbnail" src={props.img} alt="avatar.img"/>
```
Comment (adtm) [inline src/Card.js]: alt should be from props + better names

File: src/Home.js [hunk 4]
```diff
@@ -1,10 +1,23 @@
 import React from "react";
 import './App.css';
+import Card from "./Card";
+import posts from "./Data/posts";
+
+function createCard(post){
+  return (
```
Comment (adtm) [inline src/Home.js]: formatting.

**Review by adtm — CHANGES_REQUESTED** [review]
Can you add a screenshot how it looks?
Overall, you did the task, congrats :) 🎉 , now let's fix the details.

File: src/App.css [hunk 5]
```diff
@@ -21,3 +21,26 @@ nav{
...
+  box-shadow:0 2px rgb(73, 68, 68);
+  text-align: left;
+  display: flex;
+  flex-direction: row;
+}
+.right-column{
+  padding:10px;
+}
+.thumbnail {
+width: 400px;
```
Comment (adtm) [inline src/App.css:43]: nit: would suggest to fix formatting.

File: src/Card.js (hunk 3 above)
Comment (adtm) [inline src/Card.js]: probably would name like `imageSource`

Comment (adtm) [inline src/Card.js]: indentations
```
<div>
 <img>
```

Comment (Miglemeta) [reply src/Card.js]: could you, please, explain this one more? how alt should be from props? also if I name it better, I should name each picture separately then?

Comment (adtm) [reply src/Card.js]: Teoriškai jei nuotraukoje yra šuo, tai rašai alt="dog", ne "dog.img", plius pas tave visos nuotraukos turi dabar tą patį alt.

Comment (adtm) [reply src/Card.js]: tai kadangi kiekviena nuotrauka yra unikali, laikyk json'e tekstą papildomą "imageName", kurį naudosi kaip "alt" tekstą.

**Pull request merged**